docs/UserGuide.md

```diff
@@ -205,13 +206,19 @@ Format: `deletebytag -c TAG`
 * Deletes all contacts with a specific tag.
 * The tag must be a valid tag.
 
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+Since all contacts with the specified tag will be deleted permanently, we recommend you to list out 
+all those contacts first to make sure you are aware of what contacts will be deleted by using `find -c t/TAG`. 
+</div>
+
 Examples:
-* `deletebytag -c computing` deletes all contacts tagged under computing
 
-1. Entering the above example into the user input panel will delete all the contacts tagged under computing.
-![delete-by-tag-success](images/ug-images/contactsBeahviourImages/deletebytag-beofre-behaviour.png)
+* Entering `deletebytag -c computing` into the user input panel will delete all the contacts tagged under computing.
+
+![delete-by-tag-success](images/ug-images/contactsBeahviourImages/deletebytag-before-behaviour.png)
```

Comment:
i just realised there's a typo here but it looks like the directory has a typo as well (since the image still shows)